community/modules/scheduler/htcondor-access-point/main.tf

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  # This label allows for billing report tracking based on module.
+  labels = merge(var.labels, { ghpc_module = "htcondor-access-point" })
+}
+
+locals {
+  network_storage_metadata = var.network_storage == null ? {} : { network_storage = jsonencode(var.network_storage) }
+  oslogin_api_values = {
+    "DISABLE" = "FALSE"
+    "ENABLE"  = "TRUE"
+  }
+  enable_oslogin_metadata = var.enable_oslogin == "INHERIT" ? {} : { enable-oslogin = lookup(local.oslogin_api_values, var.enable_oslogin, "") }
+  metadata                = merge(local.network_storage_metadata, local.enable_oslogin_metadata, var.metadata)
+
+  name_prefix = "${var.deployment_name}-ap"
+
+  example_runner = {
```

Comment:
Why does this runner need to be added?

Reply:
The most immediate explanation is that we cannot combine "use" matching of variables with explicit runners in the blueprint. Since our tutorials expect a helloworld.sub file, I want it to remain and this is a cheap/easy approach.

We could make it optional if it's an immediate issue. Obviously, it might be nice to have a method by which to "append_to_use" or "prepend_to_use" and it could be in the blueprint. But we don't have it now.